lib/IRGen/GenStruct.cpp

```diff
@@ -680,6 +720,12 @@ class ClangRecordLowering {
 
   const TypeInfo *createTypeInfo(llvm::StructType *llvmType) {
     llvmType->setBody(LLVMFields, /*packed*/ true);
+    if (SwiftType.getStructOrBoundGenericStruct()
+            ->isCxxNotTriviallyCopyable()) {
+      return ClangRecordAddressOnlyTypeInfo::create(
+          FieldInfos, llvmType, TotalStride, std::move(SpareBits),
```

Comment:
I don't think we can assume any spare bits in a non-trivial C++ type.

Reply:
Good point.

Reply:
I had to add this back to avoid an assertion. I will investigate further.

Reply:
Nevermind, I fixed it.